models/biocro/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# ----------------------------------------------------------------------
+# BUILD PECAN FOR BIOCRO
+# ----------------------------------------------------------------------
+FROM pecan/models:${IMAGE_VERSION}
```

Comment:
I am not able to build image from this dockerfile. Should there be an IMAGE_VERSION ARG variable set up before this step?

`ARG IMAGE_VERSION='develop'`

Reply:
doh, correct.